### ISSUE TITLE
Apostles as a group?

### ISSUE THREAD

**Issue by dlschwartz** [srophe/srophe-app-data#1057]
person/1605 is "apostles". The record lists 15 entries in BHS. Presumably these are for 15 different apostles. Should each of these have their own person record/URI? I would argue yes. What do you think?

**Comment by dlschwartz** [comment]
Of course, some do have their own record, but not all. I think we want to create these. 

Also, compare person/1605; person/1606; person/1607; person/1608; person/1972; and person/2029.

**Comment by wlpotter** [comment]
#584 is also related to this one, and has a few notes on which groupings appear in which person records (see especially [this comment](https://github.com/srophe/srophe-app-data/issues/584#issuecomment-1016637332).

From what I remember of BHSE data, and also thinking about commemorations in manuscript data, I think it would be useful to distinguish the following:

- "Apostles", as a generic person group
- "The Twelve Apostles", as a smaller person group
- "The 72 Apostles", as another person group (cf. https://github.com/srophe/srophe-app-data/issues/584#issuecomment-240102261)
- We probably do what records for individual apostles, if we don't already have them. Though I think we have records for all the ones who are separately commemorated in BHSE.

There may be other attested groupings we might want to consider retaining/creating

**Comment by dlschwartz** [comment]
See also person/1791
